sanic/config.py

```diff
@@ -155,3 +136,61 @@ def strtobool(val):
         return False
     else:
         raise ValueError("invalid truth value %r" % (val,))
+
+
+# Is in Sanic any better place where to keep this ???
+
+def load_module_from_file_location(*args, **kwargs):
```

Comment:
Small change, can you prefix it: `_`

Reply:
???

Reply:
`load_module_from_file_location` => `_load_module_from_file_location`

Make it more obvious it is not meant for public consumption.

Reply:
I withdraw this comment.